### ISSUE TITLE
what about webcam

### ISSUE THREAD

**Issue by RealR3LOAD** [sharkyh20/KinectToVR#46]
well kinects are pretty expensive now (depends on country). Does kinecttovr works on webcam cuz Driver4vr seems to work- i did not try i just saw on vid-. İf it doesnt do you have any alternatives for a webcam

**Comment by HimbeersaftLP** [comment]
No, it uses the Kinect SDK which uses the Kinect's depth sensor to estimate your pose.
